File: j2k/tests/testData/ast/issues/kt-807.java
import java.io.*;

class FileRead {
    public static void main(String args[]) {
        try {
            FileInputStream fstream = new FileInputStream("textfile.txt");
            DataInputStream in = new DataInputStream(fstream);
            BufferedReader br = new BufferedReader(new InputStreamReader(in));
            String strLine;
            while ((strLine = br.readLine()) != null) { System.out.println (strLine); }
            in.close();
        } catch (Exception e) {
            System.err.println("Error: " + e.getMessage());
        }
    }
}

File: j2k/tests/testData/ast/issues/kt-807.kt
import java.io.*

class FileRead {
    class object {
        public fun main(args: Array<String>) {
            try {
                val fstream = FileInputStream("textfile.txt")
                val `in` = DataInputStream(fstream)
                val br = BufferedReader(InputStreamReader(`in`))
                val strLine: String
                while ((strLine = br.readLine()) != null) {
                    System.out.println(strLine)
                }
                `in`.close()
            } catch (e: Exception) {
                System.err.println("Error: " + e.getMessage())
            }

        }
    }
}

fun main(args: Array<String>) = FileRead.main(args)